pr_check.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Export vars for helper scripts to use
+# --------------------------------------------
+# name of app-sre "application" folder this component lives in; needs to match for quay
+export COMPONENT="subscription-inventory-ui"
+# IMAGE should match the quay repo set by app.yaml in app-interface
+export IMAGE="quay.io/cloudservices/subscription-inventory-ui"
+export WORKSPACE=${WORKSPACE:-$APP_ROOT} # if running in jenkins, use the build's workspace
+export APP_ROOT=$(pwd)
+# export NODE_BUILD_VERSION=15
+COMMON_BUILDER=https://raw.githubusercontent.com/RedHatInsights/insights-frontend-builder-common/master
+
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+#IQE_PLUGINS="host_inventory"
+#IQE_MARKER_EXPRESSION="smoke"
+#IQE_FILTER_EXPRESSION=""
```

Comment:
I'm sure there's a good reason why these are commented out, just figured I ask why

Reply:
I commented them out because I knew they weren't applicable for our application and didn't know what was. 

For now I think we should leave that commented out and once we have the pipeline in place a little more we can get with QE to figure out what to do about those